watertap/unit_models/crystallizer.py

```diff
@@ -558,6 +593,20 @@ def eq_crystallizer_height_constraint(b):
                 a + b + ((a - b) ** 2 + eps**2) ** 0.5
             )
 
+        # 13. Energy available from the vapor
+        @self.Constraint(doc="Thermal energy in the vapor")
+        def eq_vapor_energy_constraint(b):
+            return b.energy_from_vapor == (
+                b.properties_vapor[0].flow_mass_phase_comp["Vap", "H2O"]
+                * (
+                    b.properties_pure_water[
+                        0
+                    ].dh_vap_mass_solvent  # Latent heat from the vapor
```

Comment:
If the two streams are the same, then these cancel out, right?

Reply:
As explained above, the actual available heat from the vapor includes the latent heat from condensation and the sensible heat from the superheated steam, although the latter is much smaller than the former.

Reply:
Got it. Makes sense with the above clarification.

Reply:
> It's a good idea to report those in the model results.

Looking at the reporting done at the flowsheet level, I didn't see any usage of the unit model's `report` method, and given your comment here about reporting in model results, I wonder if you should add this to the unit model's `_get_performance_contents` method which is essentially part of what gets called when invoking `my_unit.report()`.